index.html

```diff
@@ -123,6 +123,42 @@
         </div>
       </ul>
     </section>
+    <section class="Earnings__right-side">
+      <dl class="Earnings__right-sign-up">
+        <dt class="Earnings__right-title">Sign up for an account</dt>
+        <dd class="Earnings__right-text">Send, spend and save smarter</dd>
+      </dl>
+        <div class="Earnings__right-button">
+          <button class="Earnings__rigth-google" type="button" aria-label="Sign up with Google">
+            <img class="Earnings__rigth-img" src="/image/Google.png" alt="Google Logo">Sign Up with Google
+          </button>
+          <button class="Earnings__rigth-apple" type="button" aria-label="Sign up with Apple">
+            <img class="Earnings__rigth-img" src="image/Apple logo.png" alt="Apple Logo">Sign Up with Apple
+          </button>
+        </div>
+        <p class="Earnings__rigth-option">or with e-mail</p>
+        <form class="Earnings__rigth-form" aria-label="form-title">
+          <div class="Earnings__rigth-names">
+            <input class="Earnings__right-imput" type="text" id="first-name" placeholder="First Name"required aria-label="First Name">
+            <input class="Earnings__right-imput" type="text" id="last-name" placeholder="Last Name"required aria-label="Last Name">
+          </div>
+          <div class="Earnings__rigth-group">
+            <input class="Earnings__right-imput" type="email" id="email"placeholder="example@email.com" required aria-label="First Name">
```

Comment:
¿Por que ese aria label? 😆 
```suggestion
            <input class="Earnings__right-imput" type="email" id="email" placeholder="example@email.com" required aria-label="Email">
```